app/models/product.rb

```diff
@@ -0,0 +1,3 @@
+class Product < ApplicationRecord
+    validates_presence_of :title, :ingredients, :price
```

Comment:
Perhaps validate that the price is an integer?

Reply:
please elaborate. do you guys want us to change the test to look for integer or validate integer it the models? in that case how do we do that?